### ISSUE TITLE
Additional contributors?

### ISSUE THREAD

**Issue by jacek-jablonski** [joelshepherd/tabliss#562]
Hi,
I've noticed that the project is pretty much dead right now. It would be a pity to abandon such a good project.
Would you mind @joelshepherd adding some contributors for accepting and reviewing PRs and fixing bugs?

**Comment by RobbieLD** [comment]
I agree, it's a nice project and a shame to let it die when there's a lot of people who are happy to contribute and make PRs. Maybe we could fork it and merge in the existing PRs and make a tabliss2.

**Comment by ahmedwael216** [comment]
It has been a while from the last commit, are you sure you don't need more people in the team?
